apps/glusterfs/app_volume.go

```diff
@@ -95,6 +95,18 @@ func (a *App) VolumeCreate(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	for _, volumeOption := range msg.Options {
+		switch volumeOption {
+		case "encryption":
```

Comment:
@obnoxxx fine.  I started  #171 by a thought of  having  free flow options from user to gluster via heketi, then by following other comments, I got a feel that we have to wrap it and develop a logic in heketi based on that.   'encryption' option was proposed in this PR by following this. I would say the current patch does have a whitelist, ie if the option does not fall into the known options we return with an error. I could see this logic as a whitelist. Whitelist or further case analysis ( using `switch`) was performed in heketi to bring  additional intelligence on these options and also to filter  different options provided by user,  for eg# if we already know there are issues in gluster when using using few options together, we can return with an error. But again, if everyone think we can do passthrough of options without wrapping or without any case analysis, I am fine with this. I will change the PR on this regard.